addons/docs/src/frameworks/common/makePreset.ts

```diff
@@ -2,10 +2,13 @@ import fs from 'fs';
 import * as common from './preset';
 
 const makePreset = (framework: string) => {
-  const frameworkConfig = `${__dirname}/../../../${framework}/config.js`;
-  const preConfig = fs.existsSync(frameworkConfig) ? [frameworkConfig] : [];
+  const docsConfig = [`${__dirname}/config.js`];
```

Comment:
@igor-dv if i wanted to somehow pass the `framework` string into this file, would that be possible?